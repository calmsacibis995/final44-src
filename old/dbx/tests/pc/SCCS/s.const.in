h18965
s 00010/00000/00000
d D 5.1 90/06/25 15:36:50 bostic 1 0
c date and time created 90/06/25 15:36:50 by bostic
e
u
U
t
T
I 1
whatis PI
whatis N
whatis MESSAGE
whatis CH
whatis p.R
print "p.R =", p.R
print "CH =", CH
print "MESSAGE =", MESSAGE
print "N =", N
print "PI =", PI
E 1
