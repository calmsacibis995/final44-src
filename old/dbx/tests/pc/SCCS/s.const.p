h25642
s 00020/00000/00000
d D 5.1 90/06/25 15:36:55 bostic 1 0
c date and time created 90/06/25 15:36:55 by bostic
e
u
U
t
T
I 1
program consttest (input, output);
const
    PI = 3.14159;
    N = 10;
    MESSAGE = 'this is a test';
    CH = 'x';
type
    Color = (RED, BLUE, GREEN);

procedure p ;
const
    R = RED;
type
    T = integer;
var i : integer;
begin
end;

begin
end.
E 1
