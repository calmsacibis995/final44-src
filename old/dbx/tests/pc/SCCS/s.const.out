h22274
s 00012/00000/00000
d D 5.1 90/06/25 15:36:51 bostic 1 0
c date and time created 90/06/25 15:36:51 by bostic
e
u
U
t
T
I 1
reading symbolic information ...
const PI = 3.14159;
const N = 10;
const MESSAGE = 'this is a test';
const CH = 'x';
const R = RED;
p.R = RED 
CH = 'x' 
MESSAGE = 'this is a test' 
N = 10 
PI = 3.14159 

E 1
